### DISCUSSION TITLE
#60 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by bambalados** [tog-ETH-er/togETHer#45] [General]
Agenda:

- Community Updates
- Fireside Chat

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/udj-mzng-dtf

**Comment by bambalados** [comment]
### CTN

- CTN21 - ETH Shanghai Upgrade  (b站链接：https://b23.tv/AcjzT2p)

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

4个PR #946 ～ #949

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《基础密码学系列课程3: 承诺、零知识证明、密钥协商》by lynndell
  - p1 https://www.youtube.com/watch?v=SnoNw0EV8Mo&list=PLgPVMJY4tnFMynvSMXsSLcnBG0MRf-Iir&index=4
  - p2 https://www.youtube.com/watch?v=CBNs-CJYuWs&list=PLgPVMJY4tnFMynvSMXsSLcnBG0MRf-Iir&index=5
 
- 《基础密码学系列课程4: 承诺、零知识证明、BulletProof范围证明、Diffie-Hellman密钥协商》by lynndell
  - p1 https://www.youtube.com/watch?v=81HiBycoB2M&list=PLgPVMJY4tnFMynvSMXsSLcnBG0MRf-Iir&index=6
  - p2 https://www.youtube.com/watch?v=UV0z_p1hEWQ&list=PLgPVMJY4tnFMynvSMXsSLcnBG0MRf-Iir&index=7

**Comment by liushooter** [comment]
## rebase

- web3 极客日报，最近 14 期 http://daily.rebase.network/
- 10 份招聘 https://github.com/rebase-network/who-is-hiring/issues

### 分享

《Web3 搜索为何如此重要》
p1: https://www.bilibili.com/video/BV1zk4y1i7Je/
p2: https://www.bilibili.com/video/BV1Eh411371D/

**Comment by Stamp9** [comment]
## 原语里弄

研究课题进行中。

**Comment by muziknozik** [comment]
Plancker：

**以太坊上海升级活动**
    3.29/3.31两场研讨会
        
        1⃣️用费曼学习法读以太坊白皮书
        📷观看回放：[https://youtu.be/MXnYcH-3qXQ](https://youtu.be/MXnYcH-3qXQ)
        2⃣️上海升级和以太坊Roadmap
        📷观看回放：[https://youtu.be/xXP3FC4AiEc](https://youtu.be/xXP3FC4AiEc)
        
4.1/2 Online Summit，物料整理后续宣发ing
        
        运营小组参与到 4.1/2 Online Summit 的筹备和统筹全流程。社区成员Pablo host了第四模块「以太坊的运用与未来」，York代表社区做模块开场演讲，K主讲了ReFi和DeSci：加密公共物品的现实场景，bob主讲了Gitcoin 如何支持Public Goods。

4.8/9/12会陆续有活动继续

**Comment by 0x68616f79756e** [comment]
ETHPlanet
ETHBeijing活动协助
EDCON筹备

**Comment by Riely007** [comment]
### 翻译小组

- 官网翻译日常进行中
